app/controllers/katello/concerns/organizations_controller_extensions.rb

```diff
@@ -53,16 +47,12 @@ def update
         super # we run super first here so that flash messages won't be in a confusing order
```

Comment:
Now that we don't have multiple flash messages any more, this `super` can be moved to the end of the method and the comment removed.